core/o11y/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "near-o11y"
+description = "Observability helpers for the near codebase"
```

Comment:
please also add README.md file to this directory - and mention more details (of what we have here and what should belong here in the future)

And also maybe a code example on how to use o11y:

    let env_filter = EnvFilter::from_default_env().add_directive(LevelFilter::INFO.into());
    near_o11y::with_default_subscriber(env_filter, || {
    YOUR CODE
    });